inc/lock.class.php

```diff
@@ -588,7 +588,7 @@ static function getLocksQueryInfosByItemType($itemtype, $baseitemtype) {
     * @since 0.85
```

Comment:
Can be removed too.a

Reply:
These were previously removed, but I readded them because I didn't realize Lock has no parent classes so it doesn't inherit them from CommonDBTM. It should stay as a `see` tag for now I think.